src/styles/styles.css

```diff
@@ -1,221 +1,17 @@
-/* To Do - Remove any unncessary CSS & Refactor */
```

Comment:
What is the purpose of this `styles.css` files? It looks like this is base styles. Typically a `styles.css` is the name of the bundled css file. I would suggest changing this to `base.scss` or something along those lines, and reserve the `styles.css` name for the bundled styles file.